src/util/native_fs_utils.js

```diff
@@ -452,7 +452,7 @@ function validate_bucket_creation(params) {
     }
 }
 
-async function config_file_exists(fs_context, config_path, use_lstat) {
+async function is_path_exists(fs_context, config_path, use_lstat) {
```

Comment:
whilke on it, can you please add doc for the typescript?

Reply:
Missed the doc for the typescript

Reply:
@liranmauda doc already added for this method in my previous commit.